File: Easy/PalindromicInteger.cpp
bool solve(int num) {

    if(num == 1)
        return true;

    vector<int> check;

    while(num!=0){
        check.push_back(num%10);
        num = num/10;
    }

    for(int j = 0; j < check.size()/2; j++){  
        if(check[j] != check[(check.size()-(j+1))])
            return false;
    }

    return true;
}

File: Easy/PalindromicInteger.js
class Solution {
    solve(n) {
        var s = n.toString();
        var l = s.length;
        for (var i = 0; i < l / 2; i++) {
            if (s[i] != s[l - i - 1]) {
            return false;
            }
        }
        return true;
    }
}